crates/oct-cli/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod oct_ctl_sdk;
```

Comment:
Move it as `mod oct_ctl_sdk;` (without `pub`) to `crates/oct-cli/src/main.rs` after the imports.
More info https://doc.rust-lang.org/book/ch07-02-defining-modules-to-control-scope-and-privacy.html#modules-cheat-sheet